starter/source/model/sets/create_ellipse_clause.F

```diff
@@ -0,0 +1,191 @@
+Copyright>        GNU Affero General Public License for more details.
+Copyright>
+Copyright>        You should have received a copy of the GNU Affero General Public License
+Copyright>        along with this program.  If not, see <https://www.gnu.org/licenses/>.
+Copyright>
+Copyright>
+Copyright>        Commercial Alternative: Altair Radioss Software
+Copyright>
+Copyright>        As an alternative to this open-source version, Altair also offers Altair Radioss
+Copyright>        software under a commercial license.  Contact Altair to discuss further if the
+Copyright>        commercial version may interest you: https://www.altair.com/radioss/.
+Chd|====================================================================
+Chd|  CREATE_ELLIPSE_CLAUSE         source/model/sets/create_ellipse_clause.F
+Chd|-- called by -----------
+Chd|        HM_SET                        source/model/sets/hm_set.F    
+Chd|-- calls ---------------
+Chd|        ANCMSG                        source/output/message/message.F
+Chd|        HM_OPTION_READ_MOD            share/modules1/hm_option_read_mod.F
+Chd|        MESSAGE_MOD                   share/message_module/message_mod.F
+Chd|        SETDEF_MOD                    ../common_source/modules/setdef_mod.F
+Chd|        SUBMODEL_MOD                  share/modules1/submodel_mod.F 
+Chd|        HM_GET_INTV                   source/devtools/hm_reader/hm_get_intv.F
+Chd|        HM_GET_FLOATV                 source/devtools/hm_reader/hm_get_floatv.F
+Chd|====================================================================
+      SUBROUTINE CREATE_ELLIPSE_CLAUSE(ID,TITLE,SUB_ID,SKEW,RTRANS,
+     .               CLAUSE,IS_AVAILABLE ,LSUBMODEL,UNITAB,ISKN,IAD)
+C-----------------------------------------------
+C   ROUTINE DESCRIPTION :
+C   ===================
+C   Create PART Clause from LIST
+C------------------------------------------------------------------
+C   DUMMY ARGUMENTS DESCRIPTION:
+C   ===================
+C
+C     NAME          DESCRIPTION                         
+C
+C     CLAUSE        (SET structure) Clause to be treated
+C     IPARTM1       MAP Table UID -> LocalID
+C     JCLAUSE       parameter with HM_READER (current clause read)
+C     IS_AVAILABLE  Bool / Result of HM_interface
+C     LSUBMODEL     SUBMODEL Structure.
+C============================================================================
+C-----------------------------------------------
+C   M o d u l e s
+C-----------------------------------------------
+      USE SETDEF_MOD
+      USE SUBMODEL_MOD
+      USE MESSAGE_MOD
+      USE HM_OPTION_READ_MOD
+      USE UNITAB_MOD
+      USE MY_ALLOC_MOD
+C-----------------------------------------------
+C   I m p l i c i t   T y p e s
+C-----------------------------------------------
+#include      "implicit_f.inc"
+C-----------------------------------------------
+C   C o m m o n   B l o c k s
+C-----------------------------------------------
+#include      "com04_c.inc"
```

Comment:
not allowed anymore to use commons,

You need to pass by argument , what needed